im-commits-pydriller.py

```diff
@@ -0,0 +1,36 @@
+"""Demo designed to show information about Github repositories using PyDriller."""
+from github import Github
```

Comment:
Unused import

Reply:
Thanks for examining the code thoroughly. I will get on these changes ASAP